### PR TITLE
Bulk of new rules, mainly UAC Bypasses

### PR DESCRIPTION
**Pull request by phantinuss**
There was some overlap with existing rules. I tried to cover only new aspects. Some scenarios will now trigger two rules. But in some scenarios only the new rules  triggered, so I put them in as an addition.

### CODE REVIEW

**Comment by frack113** [comment]
Thanks for the big works.

File: rules/windows/process_creation/win_uac_bypass_ieinstal.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        IntegrityLevel:
+          - 'High'
+          - 'System'
+        ParentImage|endswith: '\ieinstal.exe'
+        Image|endswith: '\AppData\Local\Temp\IDC1.tmp\[1]consent.exe'
```
Comment (frack113) [inline rules/windows/process_creation/win_uac_bypass_ieinstal.yml:25]: Hi, it is always this Temp dir ?
to be more flexible
```yaml
Image|contains: '\AppData\Local\Temp\'
Image|endswith: 'consent.exe'
```

File: rules/windows/file_event/sysmon_uac_bypass_ieinstal.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - Unknown
+level: high
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        Image: 'C:\Program Files\Internet Explorer\IEInstal.exe'
+        TargetFilename|startswith: 'C:\Users\'
+        TargetFilename|endswith: '\AppData\Local\Temp\IDC1.tmp\[1]consent.exe'
```
Comment (frack113) [inline rules/windows/file_event/sysmon_uac_bypass_ieinstal.yml:23]: Hi, it is always this Temp dir ?
to be more flexible
```yaml
TargetFilename|contains: '\AppData\Local\Temp\'
TargetFilename|endswith: 'consent.exe'
```

**Comment by phantinuss** [comment]
Hi,

I think your suggestions make sense.

The problem is an attacker which takes the code base of UACMe can stage the files in arbitrary locations (where no elevated rights are needed). These rules primarily detect the usage of UACMe without modifications. I tried to match the technique when possible but the attack surface on UAC seems so huge that new ways to circumvent it are just around the corner.

**Comment by frack113** [comment]
like Osgood said Well.. nobody's perfect! 😆

**Pull request merged**